crates/db_views/notification/src/lib.rs

```diff
@@ -123,91 +96,42 @@ pub struct InboxCombinedViewInternal {
 
 #[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
 #[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
-#[cfg_attr(feature = "ts-rs", ts(export))]
-// Use serde's internal tagging, to work easier with javascript libraries
-#[serde(tag = "type_")]
-pub enum InboxCombinedView {
-  CommentReply(CommentReplyView),
-  CommentMention(PersonCommentMentionView),
-  PostMention(PersonPostMentionView),
-  PrivateMessage(PrivateMessageView),
-}
-#[skip_serializing_none]
-#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
-#[cfg_attr(feature = "full", derive(Queryable))]
-#[cfg_attr(feature = "full", diesel(check_for_backend(diesel::pg::Pg)))]
-#[cfg_attr(feature = "ts-rs", derive(ts_rs::TS))]
 #[cfg_attr(feature = "ts-rs", ts(optional_fields, export))]
-/// A person comment mention view.
-pub struct PersonCommentMentionView {
-  pub person_comment_mention: PersonCommentMention,
-  pub recipient: Person,
-  pub comment: Comment,
+pub struct NotificationView {
```

Comment:
I'd highly recommend using the standand in every other combined view. `NotificationCombinedViewInternal` is the DB call, and `NotificationCombinedView` is an enum of the different potential views to be consumed by the UIs.

Reply:
Changed it.